src/components/uri/Visualization.tsx

```diff
@@ -397,7 +402,20 @@ export class VisualizationWrapped extends React.Component<
         this.visualizationUri = visualizationUri;
         this.exportTitle = get(mdObject, "meta.title", "");
 
-        const { afm, resultSpec } = buildAfmResultSpec(mdObject.content, visualizationType, locale);
+        const mdObjectContent: IVisualizationObjectContent = mdObject.content;
+        const mdObjectContentProperties: IProperties | undefined =
+            mdObjectContent && mdObjectContent.properties && JSON.parse(mdObject.content.properties);
+        const config: IChartConfig = this.createVisualizationConfig();
```

Comment:
it will not work here, internally it is accessing this.state.mdObject but it is not set yet/is old here inside 'prepareDataSources'. It is stored in this.state later in resolve of this promise